observability/logging/logging-6.0/log6x-clf.adoc

```diff
@@ -0,0 +1,91 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="log6x-CLF_{context}"]
+= Configuring log forwarding
+:context: logging-6x
+
+toc::[]
+
+The `ClusterLogForwarder` CR serves as a single point of configuration for log forwarding, making it easier to manage and maintain log collection and forwarding rules.
+
+* Defines inputs (sources) for log collection
+* Specifies outputs (destinations) for log forwarding
+* Configures filters for log processing
+* Defines pipelines to route logs from inputs to outputs
+* Indicates management state (managed or unmanaged)
+
+// Needs engineering eval re applicability to 6.0
+include::modules/log-forwarding-implementations.adoc[leveloffset=+1]
+
+// This needs to be called out extensively.
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
```

Comment:
To what extent is this intend to represent a valid sample.  As-is, this is not valid and does not properly describe a CLF?

Reply:
@anpingli could you help with your input here? Thank you!

Reply:
@anpingli Could you help verify if the following sample properly describe CLF and if it is valid ? If not, could you help with your input?

```
apiVersion: observability.openshift.io/v1
kind: ClusterLogForwarder
metadata:
  name: collector
  namespace: openshift-logging
spec:                                       
  collector:
  nodeSelector:
      logcollect: run
    resources: {}
  outputs:
  - name: lokistack
    lokiStack:
      authentication:
        token:
          from: secret                
          secret:
            key: token
            name: secret-to-lokistack
      labelKeys:
      - .hostname
      - .log_type
      - .kubernetes_container_name
      - .kubernetes_namespace_name
      - .kubernetes_pod_name
      target:
        name: lokistack-sample
        namespace: openshift-logging
    tls:
      ca:
        key: ca-bundle.crt
        secret: 
          name: secret-to-lokistack 
    type: lokiStack
  pipelines:
  - name: default-before
    inputRefs:
    - infrastructure
    - application
    outputRefs:
    - lokistack
  serviceAccount:
    name: logcollector
```

Reply:
Can we use this ClusterLogForwarer CustomResourceDefinition  and explain each filed?
 https://github.com/openshift/enhancements/blob/master/enhancements/cluster-logging/logs-observability-openshift-io-apis.md#clusterlogforwarer-customresourcedefinition

Reply:
@anpingli @@r2d2rnd @jcantrill Could you please confirm if the [ClusterLogForwarer CustomResourceDefinition](https://github.com/openshift/enhancements/blob/master/enhancements/cluster-logging/logs-observability-openshift-io-apis.md#clusterlogforwarer-customresourcedefinition) can be used as is in our documentation? If not, could you share the validated (working fine) CLF that we can use for our documentation?

Reply:
This example should be modified. As bug opened, we should set the examples with limits for the collector and also recommending always setting them. The bug is https://issues.redhat.com/browse/OBSDOCS-856.

Reply:
> @anpingli @@r2d2rnd @jcantrill Could you please confirm if the ClusterLogForwarer CustomResourceDefinition can be used as is in our documentation? If not, could you share the validated (working fine) CLF that we can use for our documentation?
@smunje1 This is the enhancement document that was the initial proposal but the actual API has subtly evolved since this example.  It is  good source for an "overview" but the actual API is the final source of truth.  This [api](https://github.com/openshift/cluster-logging-operator/blob/master/docs/reference/operator/api_observability_v1.adoc) reference is generated from the API and @libander has been including it in the docs.  Additionally, we are working to provide samples that can be used.  @cahartma can you please point to relevent GISTs for the time being

Reply:
Thanks for the context. 

> Additionally, we are working to provide samples that can be used.

Just for clarification, can these samples be used as is (Copy + Paste from what is provided) in our documentation?